tests/frontend/test-class-wpseo-opengraph-image.php

```diff
@@ -117,6 +117,32 @@ public function test_add_image_as_string() {
 		$this->assertEquals( $this->sample_array(), $class_instance->get_images() );
 	}
 
+	/**
+	 * Tests the wpseo_opengraph_is_valid_image_url filter.
+	 */
+	public function test_is_valid_image_url_filter() {
+
+		$class_instance = $this->setup_class();
+		$class_instance->add_image( 'https://via.placeholder.com/350x150' );
+
+		// Without the filter, this isn't considered a valid image URL.
+		$this->assertEmpty( $class_instance->get_images() );
+
+		// Enable the filter
+		add_filter( 'wpseo_opengraph_is_valid_image_url', array( $this, 'is_valid_image_url_filter' ), 10, 2 );
```

Comment:
For this test you might consider using `__return_true`  instead of implementing `is_valid_image_url_filter `